browser/help/topics/v3-ht-annotations.md

```diff
@@ -31,8 +31,20 @@ The gnomAD v3 Hail Table annotations are defined below:
   - **bin_freq**: Bin frequencies for the age histogram. This is the number of records found in each bin.
   - **n_smaller**: Count of age values falling below lowest histogram bin edge.
   - **n_larger**: Count of age values falling above highest histogram bin edge.
-- **freq_sample_count**: A sample count per sample grouping defined in the 'freq_meta' global annotation.
-
+- **freq_meta_sample_count**: A sample count per sample grouping defined in the 'freq_meta' global annotation.
+- **vrs_version**: The Variant Representation Specification ([VRS](https://vrs.ga4gh.org/en/stable/)) [version](https://github.com/ga4gh/vrs-python/tags) that was used to compute IDs on the callset.
```

Comment:
we can remove the link to the vrs-python repo in this description now that it has been added below (in `vrs-python_version`. I also understand that the hyphen between "vrs" and "python" is following their convention, but maybe we should use only underscores here for consistency